src/styles/event-form.less

```diff
@@ -775,3 +775,7 @@ form.booking-period {
 .audience .help-block {
   font-size: small;
 }
+
+.padding-left-12-5{
```

Comment:
Dergelijke utility-classes gebruiken we elders niet. Ik zou dit hier nu niet introduceren.